scripts/tree-summarizer-run-all.sh

```
@@ -0,0 +1,187 @@
+    echo ""
+    echo "=== [$ns] ==="
+
+    args="$SUBCOMMAND $ns"
+    if [ "$SUBCOMMAND" = "query" ] && [ -n "$NODE_ID" ]; then
+        args="$args $NODE_ID"
+    fi
+    if [ -n "$VERBOSE" ]; then
+        args="$args $VERBOSE"
+    fi
+
+    if output=$("$OPENHUMAN_BIN" tree-summarizer $args 2>&1); then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)